packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-mention.spec.ts

```diff
@@ -1542,29 +1542,34 @@ describe('RichTextEditorMentionListbox', () => {
             expect(pageObject.isMentionListboxOpened()).toBeTrue();
         });
 
-        // Intermittent, see https://github.com/ni/nimble/issues/2219
-        xit('should show mention popup for multiple mention configuration elements', async () => {
+        it('should show mention popup for multiple mention configuration elements', async () => {
             await appendUserMentionConfiguration(element, [
                 { key: 'user:1', displayName: 'username1' },
                 { key: 'user:2', displayName: 'username2' }
             ]);
+
             await pageObject.setEditorTextContent('@');
-            expect(pageObject.isMentionListboxOpened()).toBeTrue();
+
             expect(pageObject.getMentionListboxItemsName()).toEqual([
                 'username1',
                 'username2'
             ]);
-            await pageObject.clickMentionListboxOption(0);
+            expect(pageObject.isMentionListboxOpened()).toBeTrue();
+
+            await pageObject.sliceEditorContent(0, 2);
```

Comment:
Another thought is `setEditorTextContent` has this wait check:

https://github.com/ni/nimble/blob/52a54083a9350199fa16a7344e83efd81d0549ce/packages/nimble-components/src/rich-text/editor/testing/rich-text-editor.pageobject.ts#L251-L255

Maybe `sliceEditorContent` should have that as well? Maybe others that manipulate the content in the page object like `replaceEditorContent`?

Reply:
As mentioned in the above thread, focusing on the `sliceEditorContent` fixes the issue. Also, did the same for other methods that manipulate the content of the editor.